File: src/test/java/LuckyFishTests/LuckyFish_negative.java
package LuckyFishTests;

import com.codeborne.selenide.Selenide;
import org.example.core.enums.Env;
import org.example.core.pages.GamesPage;
import org.example.core.pages.SprutCloud;
import org.example.core.pages.games.LuckyFish;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;

public class LuckyFish_negative {

    private SprutCloud spC;
    private GamesPage gamesPage;
    private LuckyFish luckyFish;

    @BeforeEach
    void startGame() {
        spC = new SprutCloud();
        spC.enterUserName();
        gamesPage = spC.loginInto();
        luckyFish = gamesPage.getLuckyFish_game(Env.ENV02);
    }

    @AfterEach
    void tearDown() {
        Selenide.closeWebDriver();
    }

}

File: src/test/java/LuckyFishTests/LuckyFish_negative.kt
package LuckyFishTests;

import com.codeborne.selenide.Browsers
import org.example.core.Functional.GamesPageSprut
import org.example.core.enums.Env
import org.example.core.enums.Stages
import org.example.core.pages.SC_games.LuckyFish

class LuckyFish_negative : BaseTest(Stages.STABLE, Browsers.CHROME) {

    private val luckyFish: LuckyFish = (gameList as GamesPageSprut).getLuckyFish_game(Env.ENV03) as LuckyFish

}
